tree/dataframe/inc/ROOT/RDF/RDefinePerSample.hxx

```diff
@@ -39,7 +39,7 @@ class R__CLING_PTRCHECK(off) RDefinePerSample final : public RDefineBase {
 
 public:
    RDefinePerSample(std::string_view name, std::string_view type, F expression, RLoopManager &lm)
-      : RDefineBase(name, type, /*colRegister*/ {}, lm, /*columnNames*/ {}), fExpression(std::move(expression)),
+      : RDefineBase(name, type, /*colRegister*/ {nullptr}, lm, /*columnNames*/ {}), fExpression(std::move(expression)),
```

Comment:
I'm a subsequent PR you could consider adding a static function `RColumnRegister::MakeWithoutLoopManager ()` (just with a better name) to be more explicit what `{nullptr}` actually means.

Reply:
yes, in fact this whole lifetime management + registration/deregistration with RLoopManager needs a review and overhaul, it's on my radar..